### PR TITLE
Update components to include job 

### PR DESCRIPTION
**Pull request by Amtron**

### CODE REVIEW

**Comment by DanielCollins** [comment]
that's weird. i thought i disabled that bots comments

File: python/util/templates.py [hunk 1]
```diff
@@ -70,6 +70,7 @@ def generate_react_components(self):
         user = null,
         foundDomain = false,
         domain = null,
+        job = window.job,
```
Comment (DanielCollins) [inline python/util/templates.py]: don't create a variable to just use it only once. it defeats the entire point of variables.

File: python/util/reactjs.py [hunk 2]
```diff
@@ -39,14 +39,20 @@
...
              "moment": false,
              "toastSettingsJobProductionAssignmentComments": false,
              "toastSettingsJobProductionAssignmentQuote": false,
              "$": false,
              "MERCHI": false,
-             "console": false
+             "console": false,
+             "job": false,
+             "currentUser": false,
+             "window": false,
```
Comment (DanielCollins) [inline python/util/reactjs.py]: you've only added `job` to props, so what's all this other unrelated stuff doing here?

File: python/util/templates.py (hunk 1 above)
Comment (DanielCollins) [reply python/util/templates.py]: does not appear changed

File: python/util/reactjs.py (hunk 2 above)
Comment (Amtron) [reply python/util/reactjs.py]: - PRODUCT_CHECKOUT_FORM I've imported the Address component from
- window to redraw the pdf.
- ReactDOM should be added for future development
- UTILITIES_JSX I've imported alertTemplate
- currentUser I've removed.

File: python/users.py [hunk 3]
```diff
@@ -126,6 +127,11 @@ def user_type(self):
             return "System Admin"
         return "Normal User"
 
+    def is_component_builder(self):
+        """ Return where current user is component builder """
+        return self.system_roles is not None and \
+            sdk.python.util.roles.SYSTEM_COMPONENT_BUILDER in self.system_roles
```
Comment (DanielCollins) [inline python/users.py]: I don't think any of this stuff is needed to include the job.

**Comment by DanielCollins** [comment]
restarting travis, looks like it got stuck
